styles/styles.css

```diff
@@ -0,0 +1,278 @@
+/* stylelint-disable no-missing-end-of-source-newline */
+* {
```

Comment:
please don't use global settings
lets clean padding margin when it is required

Reply:
Fixed